src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/InProcessOptions.cpp

```diff
@@ -71,6 +72,16 @@ InProcessOptions::InProcessOptions(const ConfigurationSource &configurationSourc
     const auto anonAuthSection = configurationSource.GetSection(CS_ANONYMOUS_AUTHENTICATION_SECTION);
     m_fAnonymousAuthEnabled = anonAuthSection && anonAuthSection->GetBool(CS_ENABLED).value_or(false);
 
+    const auto maxRequestBodySize = configurationSource.GetSection(CS_MAX_REQUEST_BODY_SIZE_SECTION);
```

Comment:
I'll ask you offline about when this is present, absent, etc..

Reply:
I actually think this may be trickier than I thought. Default values won't be present here.

Reply:
Fixed this now. If the section is present, it means the feature is installed at a minimum.

Reply:
So we can default it 30MB if the feature is enabled but doesn't have a value aka default.